### ISSUE TITLE
Customizable Elasticsearch configuration

### ISSUE THREAD

**Issue by afaulconbridge** [opentargets/platform#532]
Currently there is a bunch of configuration for Elasticsearch that is baked into the pipeline. 

This should be externalized into a separate file, similar to the ops and data configuration.

This is related to opentargets/platform#367 (Document elasticsearch index configuration required)

**Comment by Docproc** [comment]
I had a look at AWS' Elasticsearch Service (where they manage it for you). It seemed pretty promising, however I ran in to an error in the HPA step of the pipeline:

`elasticsearch.exceptions.AuthenticationException: TransportError(401, u'{"Message":"Your request: \'/_cluster/settings\' payload is not allowed."}')`

This was triggered by the following line in `cluster.py/get_settings`

```python
/return/self.transport.perform_request('GET', '/_cluster/settings',params=/params/)
```

after some digging it turns out that "cluster/settings" not supported: https://forums.aws.amazon.com/message.jspa?messageID=683415

This may not be the only issue with this AWS service, but I stopped at this point. Running a DIY Elasticsearch instance on AWS (tuned with the script I got from Miguel) seems to work just fine.
